src/tbb/CMakeLists.txt

```diff
@@ -53,6 +54,10 @@ if (WIN32)
     set_target_properties(tbb PROPERTIES OUTPUT_NAME "tbb${TBB_BINARY_VERSION}")
 endif()
 
+if (EMSCRIPTEN)
+    set(TBB_LIB_COMPILE_FLAGS ${TBB_LIB_COMPILE_FLAGS} -fexceptions)
```

Comment:
AFAIK, you need to pass `-fexceptions` at both compile time and link time.

Reply:
You are right. It wasn't passed before and had no effect, because of static linking of `libtbb.a`. (The tests' link flags did include `-fexceptions`.)

I added it anyway. In the future, libtbb could become dynamic on WASM.